src/common/dto/response.dto.ts

```diff
@@ -0,0 +1,11 @@
+// response.dto.ts
+export class SuccessResponseDto<T> {
+  statusCode: number;
+  data: T;
+}
+
+export class ErrorResponseDto {
+  statusCode: number;
```

Comment:
I think this status code can be extracted to a seperate type and can be extended,